packages/api/src/services/gateways/dynamo/AuditLogDynamoGateway/AuditLogDynamoGateway.ts

```diff
@@ -470,6 +470,10 @@ export default class AuditLogDynamoGateway extends DynamoGateway implements Audi
       return dynamoUpdates
     }
 
+    if (!dynamoUpdates) {
+      return dynamoUpdates
+    }
```

Comment:
Do you know if this needs to be here? We create `dynamoUpdates` as an array, and the following line handles an empty array.

Reply:
The return type has changed, and it can now return undefined